test/Crate/Test/DBAL/Platforms/CratePlatformTest.php

```diff
@@ -63,6 +63,16 @@ public function getGenerateUniqueIndexSql()
         $this->markTestSkipped('Platform does not support CREATE UNIQUE INDEX.');
     }
 
+    public function testGeneratesForeignKeyCreationSql()
+    {
+        $fk = new \Doctrine\DBAL\Schema\ForeignKeyConstraint(array('fk_name_id'), 'other_table', array('id'), '');
+    
+        $this->assertEquals(
+            $this->getGenerateForeignKeySql(),
+            $this->_platform->getCreateForeignKeySQL($fk, 'test')
+        );
+    }
+    
     public function getGenerateForeignKeySql()
     {
         $this->markTestSkipped('Platform does not support ADD FOREIGN KEY.');
```

Comment:
pls move this to the testGeneratesForeignKeyCreationSql, and remove its content and also the getGenerateForeignKeySql method